set-height-width-blazor-reportviewer.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to Set the Height and Width of the Blazor Report Viewer
+description: How to set the position, height, and width of the Blazor Report Viewer
+type: how-to
+page_title: Setting the Height and Width of the Blazor Report Viewer
+slug: set-height-width-blazor-reportviewer
+position: 
+tags: 
+ticketid: 1479639
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>14.1.20.618</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How to set the height and width of the ReportViewer component?
+
+## Solution
+The Blazor Report Viewer is nested inside a `div` element and will resize itself based on the `div`'s width and height. 
+The article [How to Use Blazor Report Viewer](https://docs.telerik.com/reporting/blazor-report-viewer-how-to-use) demonstrates one possible approach: 
```

Comment:
You can use [How to Use Blazor Report Viewer] (../blazor-report-viewer-how-to-use)  I am putting the space on purpose between [] and () because you will not see the change

Reply:
```suggestion
The article [How to Use Blazor Report Viewer](.../blazor-report-viewer-how-to-use) demonstrates one possible approach: 
```